exports/taskfiles/utils/cmake.yaml

```
@@ -157,6 +157,8 @@ tasks:
   # should be stored.
   # @param {string[]} [CMAKE_TARGETS] A list of specific targets to build instead of the default
   # target.
+  # @param {string} [CMAKE_SOURCE_DIR] Optional path, relative to SOURCE_DIR, that specifies the
+  # directory containing CMakeLists.txt to use in the generate step.
```

```suggestion
  # @param {string} [CMAKE_SOURCE_DIR=.] The path, within the extraction directory, containing the
  # project's top level CMakeLists.txt.
```

* Variables that have a default need to specify that default.
* The `[]` already indicates the variable is optional, so no need to repeat that in the description.